src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -2155,13 +2155,19 @@ void Reset_D3D_Device(bool restore_assets)
                         true,
                         true,
                         true);
-                    OSVERSIONINFOA VersionInformation;
 
-                    if (GetVersionEx(&VersionInformation)) {
+#if 0
```

Comment:
I still say we don't even need this code at all.
It says "if OS = an os Thyme will never run on, do x". So lets just ditch the whole lot instead of keeping it around for no good reason.

Reply:
For now, I would like to keep it in place to test it later again. We still have a critical issue with ALT + Tab in the game, and anything related to Direct X Reset needs to be looked at in that regard. The deactivation of the code is appropriate for now.

Note that this function is called in game.